enterprise/frontend/src/embedding-sdk/components/public/StaticDashboard/StaticDashboard.tsx

```diff
@@ -17,7 +17,7 @@ import type { DashboardId } from "metabase-types/api";
 
 const _StaticDashboard = ({
   dashboardId,
-  parameterQueryParams,
+  parameterQueryParams = {},
```

Comment:
We don't seem to need this value as passing nothing doesn't crash the dashboard, but I tested with public dashboard that for dashboard without any filters, an empty object would be passed. So I mathed that behavior.